locales/en/settings.ftl

```diff
@@ -0,0 +1,40 @@
+# Settings page
+
+settings-page-title = {-product-short-name} Settings
+
+## Breach alert preferences
+
+settings-alert-preferences-title = Breach alert preferences
+settings-alert-preferences-option-one = Send breach alerts to the affected email address
+settings-alert-preferences-option-two = Send all breach alerts to the primary email address
+
+## Monitored email addresses
+
+settings-email-label-primary = {$email} (primary)
+settings-email-list-title = Monitored email addresses
+settings-email-limit-info = Your account includes monitoring of up to {$limit} emails.
+settings-email-verification-callout = Email verification required
+settings-resend-email-verification = Resend verification email
+settings-add-email-button = Add email address
+
+# This string is shown beneath each of the user’s email addresses to indicate
+# how many known breaches that email address was found in.
+settings-email-number-of-breaches-info =
+  {
+    $breachCount ->
+        [one] Appears in {$breachCount} known breach.
+       *[other] Appears in {$breachCount} known breaches.
+  }
+
+## Deactivate account
+
+settings-deactivate-account-title = Deactivate account
+settings-deactivate-account-info = You can deactivate Firefox Monitor by deleting your Firefox account.
+settings-fxa-link-label = Go to Firefox Settings
```

Comment:
```suggestion
settings-fxa-link-label = Go to {-brand-name} Settings
```

The naming of that term was pretty poor :-( (which brand?)

Reply:
Just saw that we also have {[-brand-firefox](https://github.com/mozilla/blurts-server/blob/main/locales/en/app.ftl#L805)}. :)